src/Backend/Synapse/ToStage/ToStageImporter.php

```diff
@@ -75,7 +77,11 @@ private function getAdapter(Storage\SourceInterface $source): CopyAdapterInterfa
                 $adapter = new FromABSCopyIntoAdapter($this->connection);
                 break;
             case $source instanceof Storage\SqlSourceInterface:
-                $adapter = new FromTableCTASAdapter($this->connection);
+                if ($importOptions->getTableToTableAdapter() === SynapseImportOptions::TABLE_TO_TABLE_ADAPTER_CTAS) {
+                    $adapter = new FromTableCTASAdapter($this->connection);
+                } else {
```

Comment:
radeji bych tam dal elseif na ten druhej typ a else by mela strilet exception. Jde mi o to, ze az se do toho bude hrabat, tak aby se to nechovalo divne, ze nekdo bude blbe nastavovat ten option (je to obyc string bez validace) a ono to bude v klidu delat InsertInto jako by se nechumelilo

nebo klidne switch, je mi to burt. Nebo validaci na nastavovani toho typu. Ale toto mi prijde, ze nas to muze pripadne bolet az se do toho bude hrabat

Reply:
upravil sem